app/Http/Controllers/Three/ReviewsController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Rogue\Http\Controllers\Three;
+
+use Rogue\Models\Post;
+use Illuminate\Http\Request;
+use Rogue\Repositories\Three\PostRepository;
+use Rogue\Http\Controllers\Api\ApiController;
+use Rogue\Http\Transformers\Three\PostTransformer;
+
+class ReviewsController extends ApiController
+{
+    /**
+     * The post repository instance.
+     *
+     * @var Rogue\Repositories\Three\PostRepository
+     */
+    protected $post;
+
+    /**
+     * @var \Rogue\Http\Transformers\Three\PostTransformer
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract $posts
+     * @return void
+     */
+    public function __construct(PostRepository $post)
+    {
+        $this->post = $post;
+        $this->transformer = new PostTransformer;
+
+        $this->middleware('auth:api');
+        $this->middleware('role:admin');
+    }
+
+    /**
+     * Update a post(s)'s status when reviewed.
+     *
+     * @param Request $request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function reviews(Request $request)
+    {
+        $request->validate([
+            'post_id' => 'required:exists:posts',
```

Comment:
This looks like a typo: `required:exists:posts` → `required|exists:posts`.

...but it seems like we can safely rely on the `findOrFail` below now, so I think you're safe to remove it!

Reply:
oops you're right! removed!